tools/docker/substrate-all-in-one/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+# From https://github.com/substrate-developer-hub/substrate-node-template
```

Comment:
@RafaelAPB @CatarinaPedreira Just double checking, is this still operational? Has it been tested manually with the latest published official docker image?

Reply:
@petermetz This file is not used anymore, we are now running the container only using the Dockerfile. I'm going to remove it. 
I tested this with the latest image, but the error remained (a system file error). As I needed to move forward and have time constraints I moved on without the yaml, but I can go back at a later stage to try and fix this for coherence with other test ledgers :)

Reply:
@CatarinaPedreira Totally get it regarding the time constraints, will be happy to accept extra PRs with it in the future if you ever get around to adding it, but also zero pressure of course, this is great already as it is!

Reply:
Thank you Peter :)